src/observer.js

```diff
@@ -205,8 +211,10 @@ const reactiveMixin = {
           this.__$mobRenderEnd = Date.now();
         }
       });
-      if (exception)
+      if (exception) {
+        errorsReporter.emit(exception);
```

Comment:
On the other hand, the throw doesn't add that much here, since Reactions catch all exception, the only additional value of this throw is that the error get's printed to the console